### PR TITLE
Add Class Levels page

### PR DESCRIPTION
**Pull request by mxie**
This will be referenced in the participant registration so that they'll have a
better idea as to which category they fall into.

This content is borrowed from workshop signups through BridgeTroll, e.g. https://www.bridgetroll.org/events/150/rsvps/new/learn.

### CODE REVIEW

**Comment by mxie** [comment]
@conchan Can you take a look and maybe fix the styling a bit? It's looks rather awkward currently.

class levels - railsbridge boston 2015-03-04 10-30-46

**Comment by conchan** [comment]
classlevels-small

classlevels

File: source/class-levels.html.erb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  <section class="levels level-3">
+    <h2 class="section-title">Some Rails Experience</h2>
+    <ul>
+      <li>You're comfortable using the terminal, but not necessarily a Power User.</li>
+      <li>You have a general understanding of a Rails app's structure, perhaps from a prior workshop or tutorial.</li>
+      <li>You know how to define a method in Ruby.</li>
+      <li>You have a decent handle on Ruby arrays and hashes.</li>
+    </ul>
+
+</section>
```
Comment (mborsare) [inline source/class-levels.html.erb]: indent

File: source/class-levels.html.erb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  <section class="levels level-4">
+    <h2 class="section-title">Other Programming Experience</h2>
+    <ul>
+      <li>You're proficient in another language and understand general programming concepts, like collections and scope.</li>
+      <li>You're new to Ruby and Rails</li>
+      <li>You might be familiar with version control and basic web architecture</li>
+    </ul>
+  </main>
```
Comment (mborsare) [inline source/class-levels.html.erb]: switch with line 50 and correct indentation

File: source/class-levels.html.erb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+  <section class="levels level-3">
+    <h2 class="section-title">Some Rails Experience</h2>
+    <ul>
+      <li>You're comfortable using the terminal, but not necessarily a Power User.</li>
+      <li>You have a general understanding of a Rails app's structure, perhaps from a prior workshop or tutorial.</li>
+      <li>You know how to define a method in Ruby.</li>
+      <li>You have a decent handle on Ruby arrays and hashes.</li>
+    </ul>
+
```
Comment (kylefiedler) [inline source/class-levels.html.erb]: You have an empty line here but not in the sections above. Just be constant.

File: source/stylesheets/_class-levels.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+    &::before {
+      @include margin(null 0.5em null -0.75em);
+      @include size(0.25em);
+      background-color: $gray-light;
+      border-radius: 50%;
+      content: "";
+      display: inline-block;
+      vertical-align: middle;
+    }
```
Comment (mborsare) [inline source/stylesheets/_class-levels.scss]: So classy with the hanging bullets!

File: source/class-levels.html.erb [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+</section>
+  <section class="levels level-4">
+    <h2 class="section-title">Other Programming Experience</h2>
+    <ul>
+      <li>You're proficient in another language and understand general programming concepts, like collections and scope.</li>
+      <li>You're new to Ruby and Rails</li>
+      <li>You might be familiar with version control and basic web architecture</li>
+    </ul>
+  </main>
+
```
Comment (kylefiedler) [inline source/class-levels.html.erb]: Same empty line thing here.

File: source/stylesheets/_class-levels.scss [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+@each $level in $levels {
+  $number: index($levels, $level);
+
+  .#{$level}::before {
+    border-color: lighten($gray, 50 / $number);
+    color: lighten($gray, 50 / $number);
+    content: "#{$number}";
+  }
+}
```
Comment (mborsare) [inline source/stylesheets/_class-levels.scss]: fancy!

File: source/stylesheets/_class-levels.scss (hunk 4 above)
Comment (conchan) [inline source/stylesheets/_class-levels.scss]: Gotta do it right!

**Comment by mborsare** [comment]
:sheep:

File: source/stylesheets/_class-levels.scss (hunk 6 above)
Comment (mxie) [inline source/stylesheets/_class-levels.scss]: Huh. This _is_ fancy!

**Pull request merged**